File: 1816-truncate-sentence/1816-truncate-sentence.js
/**
 * @param {string} s
 * @param {number} k
 * @return {string}
 */
var truncateSentence = function(s, k) {
    let count = 0;
    let str = "";

    for(let i = 0; i < s.length; i++) {
        if (s[i] === ' ') {
            count++;
            if (count === k) {
                return str;
            };
        }
        str += s[i];
    }
    return s;
};

File: 1816-truncate-sentence/1816-truncate-sentence.cpp
class Solution {
public:
    string truncateSentence(string s, int k) {
        int count = 0;
        string str = "";

        for(int i = 0; i < s.size(); i++) {
            if (s[i] == ' ') {
                count++;
                if (count == k) {
                    return str;
                };
            }
            str += s[i];
        }
        return s;
    }
};